h2gis-utilities/src/main/java/org/h2gis/utilities/JDBCUtilities.java

```diff
@@ -158,6 +158,17 @@ public static String getFieldName(DatabaseMetaData meta, String table, int field
         }
         return fieldNameList;
     }
+    
+    /**
+     * Fetch the row count of a table.
+     * @param connection Active connection.
+     * @param location Table location
+     * @return Row count
+     * @throws SQLException If the table does not exists, or sql request fail.
+     */
+    public static int getRowCount(Connection connection, TableLocation location) throws SQLException {
+        return getRowCount(connection, location.toString());
```

Comment:
The `TableLocation` string representation depends of the database type (H2 or not) so you should use `TableLocation.toString(boolean isH2)`